vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -59,6 +64,12 @@ public String space(@PathVariable("id") String id, Model model) {
         if (space.getSpaceStatus() == null || space.getSpaceStatus().equals(SpaceStatus.PUBLISHED)
                 || authenticationFacade.getAuthenticatedUser() != null) {
             IExhibition exhibition = exhibitManager.getStartExhibition();
+            List<ExhibitionAboutPage> aboutPageList = aboutPageManager.findAll();
+            ExhibitionAboutPage exhibitionAboutPage = aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
+            if(exhibitionAboutPage != null) {
```

Comment:
can that ever be the case?

Reply:
When the about page is not configured exhibitionAboutPage can be null.

Reply:
but line 68 initializes the variable in that case, doesn't it?

Reply:
Sorry I meant exhibitionAboutPage.getTitle() and exhibitionAboutPage.getAboutPageText() will be null if the about page is not configured so I thought there is no need to add that to the model attributes. But I am checking null anyway in the html so I can remove the null check here, I will make the change.

Reply:
Yes my bad. I see what you mean. I have made the change